bindings/pyroot/pythonizations/python/ROOT/_pythonization/_rdataframe.py

```diff
@@ -417,3 +417,175 @@ def pythonize_rdataframe(klass):
                                       HistoProfileWrapper,
                                       model_class)
         setattr(klass, method_name, getter)
+
+
+from . import pythonization
+from .._numbadeclare import _NumbaDeclareDecorator
+import re
+import warnings
+import numpy as np
+
+# Pythonizer 
+# def pythonize_define():
+    # @pythonization('RDataFrame', ns = 'ROOT')
+    # def pythonizer_rdf(klass):
+    #     klass._OriginalDefine =  klass.Define
+    #     klass.Define = _PyDefine
+
+
+@pythonization("RInterface<", ns="ROOT::RDF", is_prefix=True)
+def pythonize_rdataframeDefine(klass):
+    klass._OriginalDefine =  klass.Define
+    klass.PyDefine = _PyDefine
+
+# def pythonize_filter():
+    # @pythonization('RDataFrame', ns = 'ROOT')
+    # def pythonizer_rdf(klass):
+    #     klass._OriginalFilter =  klass.Filter
+    #     klass.Filter = _PyFilter
+
+@pythonization("RInterface<", ns="ROOT::RDF", is_prefix=True)
+def pythonize_rdataframeFilter(klass):
+    klass._OriginalFilter =  klass.Filter
+    klass.Filter = _PyFilter
+
+
+def _PyFilter(rdf, func, funcargs = {}, filter_name = ""):
```

Comment:
We need some docs for these helpers for filter and define. Please see how docstrings are done for other helper functions in this module or in other pythonization modules.